--- compat_dcache.h	2014-09-07 18:33:57.939430889 +0700
+++ compat_dcache.patch.h	2014-09-07 18:34:40.186450921 +0700
@@ -53,6 +53,8 @@
  */
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 11, 0)
 #define compat_d_count(dentry) d_count(dentry)
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(3, 10, 0)
+#define compat_d_count(dentry) d_count(dentry)
 #elif LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 38)
 #define compat_d_count(dentry) dentry->d_count
 #else
